add-ons/istio/istio.addon

```diff
@@ -0,0 +1,37 @@
+# Name: istio
+# Description: Deploys Latest Istio
+# Url: https://github.com/istio/istio
+
+# From istio doc
```

Comment:
@kameshsampath , Could we create a new project for istio like we have for Eclipse che https://github.com/minishift/minishift-addons/blob/master/add-ons/che/che.addon#L12.

This seems to be easy to undeploy the service also https://github.com/minishift/minishift-addons/blob/master/add-ons/che/che.addon.remove#L12

Reply:
it does automatically @budhrg  , a project called istio-system will be created automatically

Reply:
:+1: